packages/components/src/font-size-picker/next.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { FontSizeControl } from '@wp-g2/components';
+
+/**
+ * Internal dependencies
+ */
+import { __unstableWithNext as withNext } from '../__next/context';
+
+export function withNextComponent( current ) {
+	return withNext( current, FontSizeControl, 'WPComponentsFontSizePicker' );
```

Comment:
I wonder if for something like a FontSizePicker we really need to keep two versions. I guess in this case would be ok if we just deleted the current component and used the new one?

Reply:
That's up to us really. It looks like `FontSizePicker` is only really used in one spot, that being the `hooks` typography tools.

The thing that is uncertain is if it's used by 3rd party blocks. In that case, I feel like it may be safer to...

1. Start with having both
2. Ensure it's stable
3. Create some documentation for 3rd party devs regarding this migration (context/adapter). Announce this.
4. See if there's any pushback
5. If there are many people who want to keep the old `FontSizePicker`, provide 3rd party devs with the ability to "opt-out" (with maybe deprecation notices)
6. Wait a bit
7. Remove old implementation

☝️ Repeat for components as needed